src/installer/tests/HostActivation.Tests/PortableAppActivation.cs

```diff
@@ -484,13 +486,10 @@ public void AppHost_CLI_FrameworkDependent_MissingRuntimeFramework_ErrorReported
                     .Start();
 
                 var result = command.WaitForExit(true)
-                    .Should().Fail();
-
-                result.And.HaveStdErrContaining($"- https://aka.ms/dotnet-core-applaunch?{expectedUrlQuery}");
-                if (expectedUrlParameter != null)
-                {
-                    result.And.HaveStdErrContaining(expectedUrlParameter);
-                }
+                    .Should().Fail()
+                    .And.HaveStdErrContaining($"- https://aka.ms/dotnet-core-applaunch?{expectedUrlQuery}")
+                    .And.HaveStdErrContaining(expectedStdErr)
+                    .And.ExitWith(expectedErrorCode);
```

Comment:
I don't think this will work properly on some (Unix) systems where the exit codes are 0-255